tilty/tilt_device.py

```diff
@@ -38,24 +38,29 @@ def stop(self) -> None:
         LOGGER.debug('Stopping device socket')
         blescan.hci_disable_le_scan(self.sock)
 
-    def scan_for_tilt_data(self) -> dict:
+    def scan_for_tilt_data(self) -> list:
         """ scan for tilt and return data if found """
 
-        data = {}
+        data = []
         LOGGER.debug('Looking for events')
         for beacon in blescan.get_events(self.sock):
-            if beacon['uuid'] in constants.TILT_DEVICES:
-                data = {
-                    'color': constants.TILT_DEVICES[beacon['uuid']],
+            uuid = beacon.get('uuid')
+            if uuid is None:
+                continue
+            color = constants.TILT_DEVICES.get(uuid)
+            if color:
+                data.append({
+                    'color': color,
                     'gravity': float(beacon['minor']/1000),
                     'temp': beacon['major'],
                     'mac': beacon['mac'],
                     'timestamp': datetime.now().isoformat(),
-                }
+                    'uuid': uuid
```

Comment:
You probably realized this from the discussion after the fact but there's no real benefit to storing color _and_ uuid since theyre 1:1

later on in other parts of the module if you need the uuid you can look it up by color. could probably make a wrapper to make it easier:

```
>>> from tilty import constants
>>> [dev for dev in constants.TILT_DEVICES.items() if dev[1] == 'Black'][0][0]
'a495bb30c5b14b44b5121370f02d74de'
```